cypress/support/step-definitions/select-steps.js

```diff
@@ -92,3 +92,26 @@ When('Type {string} text into input and select the value', (text) => {
 When('Type {string} text into input and select the value into iFrame', (text) => {
   selectInputNoIframe().type(`${text}{downarrow}{enter}`);
 });
+
+Then('Select is transparent', () => {
+  select().parent().should('have.css', 'background-color', 'rgba(0, 0, 0, 0)')
+    .and('have.css', 'border-left-color', 'rgba(0, 0, 0, 0.85)')
+    .and('have.css', 'border-right-color', 'rgba(0, 0, 0, 0.85)')
+    .and('have.css', 'border-top-color', 'rgba(0, 0, 0, 0.85)');
+});
+
+Then('Select is not transparent', () => {
+  select().parent().should('have.css', 'background-color', 'rgb(255, 255, 255)')
+    .and('have.css', 'border-left-color', 'rgb(102, 132, 145)')
+    .and('have.css', 'border-right-color', 'rgb(102, 132, 145)')
+    .and('have.css', 'border-top-color', 'rgb(102, 132, 145)');
+});
+
+Then('Select placeholder align on preview is set to {string}', (direction) => {
+  if (direction === 'left') {
+    // left is default property that's why it's absent inside class
+    select().should('have.css', 'text-align', `${direction}`);
+  } else {
+    select().should('have.css', 'text-align', `${direction}`);
```

Comment:
here should be probably 
```suggestion
    select().should('not.have.css', 'text-align', direction);
```

Reply:
in step I pass proper 'text-align', it's why I used `have.css`